### PR TITLE
[SPARK-27261][DOC] Improve app submission doc for passing multiple configs

### PR DESCRIPTION
**Pull request by sujith71955**
## What changes were proposed in this pull request?

While submitting the spark application, passing multiple configurations not documented clearly, no examples given.it will be better if it can be documented since  clarity is less from spark documentation side.
Even when i was browsing i could see few queries raised by users, below provided the reference.

https://community.hortonworks.com/questions/105022/spark-submit-multiple-configurations.html

 As part of fixing i had documented the above scenario  with an example.

## How was this patch tested?
Manual inspection of the updated document.

### CODE REVIEW

**Comment by sujith71955** [comment]
@cloud-fan @dongjoon-hyun Please let me know for your suggestions. thanks

File: docs/submitting-applications.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Some of the commonly used options are:
 * `--class`: The entry point for your application (e.g. `org.apache.spark.examples.SparkPi`)
 * `--master`: The [master URL](#master-urls) for the cluster (e.g. `spark://23.195.26.187:7077`)
 * `--deploy-mode`: Whether to deploy your driver on the worker nodes (`cluster`) or locally as an external client (`client`) (default: `client`) <b> &#8224; </b>
-* `--conf`: Arbitrary Spark configuration property in key=value format. For values that contain spaces wrap "key=value" in quotes (as shown).
+* `--conf`: Arbitrary Spark configuration property in key=value format. For values that contain spaces wrap "key=value" in quotes (as shown). Multiple configurations shall be passed individually.(e.g --conf <key>=<value> --conf <key>=<value>)
```
Comment (HeartSaVioR) [inline docs/submitting-applications.md]: nit: `key2=value2` for second option

**Review by HeartSaVioR — APPROVED** [review]
LGTM, a nit.

File: docs/submitting-applications.md (hunk 1 above)
Comment (felixcheung) [reply docs/submitting-applications.md]: also put `--conf <key>=<value> --conf <key>=<value` in backtick

**Comment by dongjoon-hyun** [comment]
Thank you for pinging me, @sujith71955 . Yes, this will be helpful. Please revise according to the previous comments from @HeartSaVioR and @felixcheung .

**Review by HyukjinKwon — APPROVED** [review]
Looks good except the nit

File: docs/submitting-applications.md (hunk 1 above)
Comment (srowen) [reply docs/submitting-applications.md]: And you're missing a space after period, period after g, and the second sentence is all parenthetical. For minor changes, please do proofread them. Just: "Multiple configurations should be passed as separate arguments. Example: `--conf key1=value1 --conf key2=value2`.

Comment (sujith71955) [reply docs/submitting-applications.md]: Thanks all for review, will update the section as per the suggestions .

**Comment by sujith71955** [comment]
All comments has been addressed, let me know for any more clarifications/suggestions. Thanks all for your valuable time.

**Review by srowen — APPROVED** [review]
Close enough

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to master.

**Pull request closed without merging**